### DISCUSSION TITLE
Prevent doubling back on mulitiple perimeters?

### DISCUSSION THREAD

**Discussion by DocRuzzy** [supermerill/SuperSlicer#1303] [Q&A]
Is there a way force perimeters to always lay down in the same direction (e.g. clockwise)?
I have a small diameter hollow vertical tube that wont print correctly because the perimeters printing doubles back rather than progressing through, this causes some gap and cooling issues. When I'm laying down a brim, it nicely keeps a single direction spiral, but with 2 or more perimeters I get the behavior below.
![Double-back perimeter](https://user-images.githubusercontent.com/67708551/121783989-750cf280-cb7f-11eb-9fe3-b85331d67ee0.gif)

**Comment by supermerill** [comment]
That's because the first one is around a hole, so it prints it in the other direction.
No setting can erase that behavior.

**Reply by DocRuzzy** [reply]
Thanks! I just made it solid with zero infill and fixed the problem!
